### DISCUSSION TITLE
Sonic

### DISCUSSION THREAD

**Discussion by JerryM33** [skyline-emu/skyline#312] [General]
![Screenshot_20220109_224926](https://user-images.githubusercontent.com/97395782/148687959-939a174c-90f4-44cf-b229-0bdf6a89cdd7.jpg)

**Comment by Bossplay228** [comment]
Что делать если я запускаю Соника но у меня темный экран, но звуки я слышу

**Reply by Allatel** [reply]
Скачай билд из ветки ftx1  графика только там

**Reply by AyderProgramic** [reply]
Какое у тебя устройство?
